client/src/docs-ui/search-bar/search-bar.style.ts

```diff
@@ -0,0 +1,93 @@
+import {MQLaptop} from "../../amplify-ui/styles/media";
+import {css} from "emotion";
+
+// we use !important to override webkit's imposed border-radius
+export const searchStyle = css`
+  width: 100%;
+
+  ${MQLaptop} {
+    width: initial;
+    margin: 0 1rem;
```

Comment:
`margin: 0 4rem` looks a little better. Let's chat on this tomorrow.

Reply:
In agreement with Jordan on this, can we make the change?